### DISCUSSION TITLE
When is next planned release?

### DISCUSSION THREAD

**Discussion by harshalgohel** [lldpd/lldpd#516] [General]
Friendly question, if I should wait for next release or backport new commits?

**Comment by vincentbernat** [comment]
> Friendly question, if I should wait for next release or backport new commits?

I can do a release soon. For example, next week if that's OK for you.

**Reply by harshalgohel** [reply]
> Friendly question, if I should wait for next release or backport new commits?
> I can do a release soon. For example, next week if that's OK for you.

Nice. That would work for me. Thanks.

**Comment by vincentbernat** [comment]
❦ 16 May 2022 01:42 -07, Harshal Gohel:

>> Friendly question, if I should wait for next release or backport new commits?
>> I can do a release soon. For example, next week if that's OK for you.
>
> Nice. That would work for me. Thanks.

It has just been released.

**Reply by harshalgohel** [reply]
This is awesome!
